### ISSUE TITLE
Order Creation: Accessibility → Headings should be dictated as "Heading"

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-ios#6065]
<img src="https://user-images.githubusercontent.com/198826/152423926-bbc18e88-524f-4e3e-bd1b-f392ceacd6bc.png" width="300">

## Steps

1. Turn VoiceOver on
2. Navigate to Products heading. 

VoiceOver should say "Products. Heading"

**Comment by wiksien** [comment]
I would like to fix this issue.

**Comment by shiki** [comment]
@wiksien Go for it! Note that this feature is hidden behind the Experimental Feature section in the Settings. So you'll have to turn that on to see this UI. 🙂 Let me know if you can't find it!

Some more info here: https://github.com/woocommerce/woocommerce-ios/pull/6060

**Comment by AE542** [comment]
Hey I just wondered if this issue has been tackled? I could add a simple accessibility hint on VoiceOver to make it work if it's not been done yet.

Thanks!

**Comment by shiki** [comment]
Hi, @AE542! Not yet. Thank you for volunteering. 

I think this is missing a [header accessibility trait](https://developer.apple.com/documentation/uikit/uiaccessibility/uiaccessibilitytraits/1620170-header).

I'll assign this issue to you. 🙂 Let me know if you need help!

**Comment by AE542** [comment]
Hi shiki, sorry I'm ashamed to say I've been having trouble even getting the app to load on my MacBook. Do you need home-brew and rbenv to get it up and running?

**Comment by shiki** [comment]
Hi, @AE542. Sorry to hear that. What sort of trouble are you running into? I don't think you'd need homebrew specifically but `rbenv` would definitely help so that you'll have the correct Ruby version. The Ruby version you would need is defined in [`.ruby-version`](https://github.com/woocommerce/woocommerce-ios/blob/trunk/.ruby-version).

**Comment by AE542** [comment]
OK I'll have a go at it tomorrow but specifically there are a lot of errors regarding pods so I'm going to deinit and reinstall them. If it doesn't work out I'll let you know. Sorry for the trouble!

**Comment by shiki** [comment]
Hi, @AE542! How are you doing? Need any help? Are you still planning on tackling this? 🙂

**Comment by AE542** [comment]
Hey Shiki, sorry I've been busy with some other work lately. I'm trying to tackle it and just trying to get the app to load still. I'll give it a shot tomorrow and if it's not playing ball I'll pass it back to you. Sorry!

**Comment by shiki** [comment]
Oh, no problem at all. 🙂 Good luck!

**Comment by rachelmcr** [comment]
Hi @AE542! Thanks for offering to work on this. I'm going to take on this issue but please let us know if we can help you get the app running locally and if there are any other issues you'd like to work on. :)

**Comment by AE542** [comment]
> Hi @AE542! Thanks for offering to work on this. I'm going to take on this issue but please let us know if we can help you get the app running locally and if there are any other issues you'd like to work on. :)

Hello Rachel and Shiki, I'm so sorry I couldn't get around to fixing this. I just had issues getting it running among other stuff going on at the moment. My apologies.

**Comment by rachelmcr** [comment]
No worries! That's understandable and we're happy to help sort out any issues with the project if you want to give it another try sometime.
